src/components/Data.js

```diff
@@ -0,0 +1,87 @@
+export const startTime = new Date(2021, 6, 6, 0, 0, 0)
+
+export const firstEvents = [
+    {
+        name: "05:00PM - Kickoff via Discord (opening-kickoff)",
+        endTime: new Date(2021, 6, 10, 18, 0, 0)
```

Comment:
This is static data why are we creating new Dates in here? Convert these all to readable Date strings instead and make them new Date when rendering data.